src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -302,4 +302,32 @@ else if (userMessagesCount != 0 && userMessagesCount == messages)
 	  return mostActiveUser;
 		  
 	}
+	
+	 /** Stores the User in Datastore. */
+	 public void storeUser(User user) {
+	  Entity userEntity = new Entity("User", user.getEmail());
+	  userEntity.setProperty("email", user.getEmail());
+	  userEntity.setProperty("aboutMe", user.getAboutMe());
+	  datastore.put(userEntity);
+	 }
+	 
+	 /**
+	  * Returns the User owned by the email address, or
+	  * null if no matching User was found.
+	  */
+	 public User getUser(String email) {
+	 
+	  Query query = new Query("User")
+		.setFilter(new Query.FilterPredicate("email", FilterOperator.EQUAL, email));
```

Comment:
This line should be indented +4 with respect to the previous line as the code here is continuation of the previous line.